app/controllers/results_submission_controller.rb

```diff
@@ -12,6 +12,28 @@ def new
     @results_validator.validate(@competition.id)
   end
 
+  def newcomer_checks
+    @competition = competition_from_params
+  end
+
+  def last_duplicate_checker_job_run
+    last_job_run = DuplicateCheckerJobRun.where(competition_id: params[:competition_id]).first
```

Comment:
I think `where(...).first` is the same as `find_by(...)`. Can you please check to see whether that respects your `default_scope`? (I'm already 90% sure that it should, just need those extra 10% for confirmation)

Reply:
Okay done